### PR TITLE
new fixes for UEFI support

### PR DESCRIPTION
**Pull request by M1cha**
This change is [Reviewable](https://reviewable.io/reviews/lkl/linux/351)

### CODE REVIEW

File: tools/lkl/lib/virtio.c [hunk 1]
```diff
@@ -549,8 +547,8 @@ int virtio_dev_setup(struct virtio_dev *dev, int queues, int num_max)
 
 	if (!lkl_is_running()) {
 		avail = sizeof(lkl_virtio_devs) - (devs - lkl_virtio_devs);
-		num_bytes = snprintf(devs, avail, " virtio_mmio.device=%d@0x%"PRIxPTR":%d",
-				     mmio_size, (uintptr_t) dev->base, dev->irq);
+		num_bytes = snprintf(devs, avail, " virtio_mmio.device=%d@%p:%d",
```
Comment (tavip) [inline tools/lkl/lib/virtio.c]: IIRC %p does not work correctly for mingw. The mingw tests seem to confirm this.

**Comment by tavip** [comment]
@lkl-jenkins: test this please

**Comment by thehajime** [comment]
thanks ! please also address the checkpatch.sh error that circleci reported.

**Comment by M1cha** [comment]
I've removed 'tools: lkl: use "%p" instead of "0x%"PRIxPTR' and added support for inttypes.h support to my libc implementation instead.

**Comment by tavip** [comment]
@lkl-jenkins: test this please

**Comment by tavip** [comment]
Reviewed 2 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/lkl/linux/351)*

**Pull request merged**